packages/patternlab/styleguide/source/assets/js/modules/stickyTOC.js

```diff
@@ -98,20 +103,15 @@ export default (function (window, document) {
           if ((heading.innerText.toLowerCase() != 'related') && (heading.innerText.toLowerCase() != 'contact')) {
             tocSectionCount++;
           }
-          else if (heading.innerText.toLowerCase() == 'contact') {
-            additionalCount++;
+          else {
```

Comment:
this else statement is redundant and can be removed

Reply:
It should be there, in case the TOC is visible it will remove the contact and remove